Strategic/Map Screen Interface Map Inventory.cpp

```diff
@@ -6227,9 +6021,6 @@ void HandleItemCooldownFunctions( OBJECTTYPE* itemStack, INT32 deltaSeconds, BOO
 
 			FLOAT newguntemperature = max(0.0f, guntemperature - tickspassed * cooldownfactor );	// ... calculate new temperature ...
 
-#if 0//def JA2TESTVERSION

```

Comment:
Change this to JA2TESTVERSION or add ITEMTEMPDEBUG def and keep the code? If we need to track item temp changes this would be useful